modules/ting_search/ting_search.module

```diff
@@ -474,31 +483,45 @@ function ting_search_search_execute($keys = NULL, $conditions = NULL) {
         }
       }
     }
-    catch (TingClientException $e) {
+    catch (\Exception $e) {
       watchdog_exception('ting_search', $e);
       $results = array();
     }
 
     // Save search result for usage by facet modules etc.
-    $ting_search_results = &drupal_static('ting_search_results');
-    $ting_search_results = $search_result;
+    $current_results = &ting_search_current_results();
+    $current_results = $search_result;
 
+    // TODO BBS-SAL: The following code either needs to updated to SAL (eg.
+    // ding_facets needs to be updated) or removed.
+    /*
     // Add support for facet api.
     if (module_exists('ding_facets')) {
-      // @todo ITK Aarhus finish implementation of facet api to use a standard
-      // facet module, that allows usage of modules from D.O. to improved facet
-      // usage.
-      $adapter = facetapi_adapter_load('ding_facets');
-      if ($adapter) {
-        $adapter->addActiveFilters($query);
-      }
+    // @todo ITK Aarhus finish implementation of facet api to use a standard
+    // facet module, that allows usage of modules from D.O. to improved facet
+    // usage.
+    $adapter = facetapi_adapter_load('ding_facets');
+    if ($adapter) {
+    $adapter->addActiveFilters($query);
+    }
     }
+     */
```

Comment:
Is there any reason to outcomment this code and remove the indentation here? If the module is not available then it is in practice dead code. I would think the initial comment was enough.

Reply:
True